<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.2.2 (Python 3.8.6 on win32)" generated="20210419 12:44:29.310" rpa="false">
<suite id="s1" name="FR1 01-01 CaptureAudit AS1" source="C:\Users\Delwin.Horsthemke\PycharmProjects\Robot_ST_Templates\TestCases\Audits\FR1_01-01_CaptureAudit_AS1.robot">
<test id="s1-t1" name="FR1_01-01_CaptureAudit_AlternateScenario1">
<kw name="Launch IsoMetrix Application">
<msg timestamp="20210419 12:44:29.813" level="FAIL">No keyword with name 'Launch IsoMetrix Application' found.</msg>
<status status="FAIL" starttime="20210419 12:44:29.813" endtime="20210419 12:44:29.813"></status>
</kw>
<tags>
<tag>behavior-driven</tag>
</tags>
<status status="FAIL" starttime="20210419 12:44:29.807" endtime="20210419 12:44:29.814" critical="yes">No keyword with name 'Launch IsoMetrix Application' found.</status>
</test>
<test id="s1-t2" name="FR1_01-01_CaptureAudit_Logged Notification">
<kw name="FR1_Audits_LoggedNotification">
<msg timestamp="20210419 12:44:29.821" level="FAIL">No keyword with name 'FR1_Audits_LoggedNotification' found.</msg>
<status status="FAIL" starttime="20210419 12:44:29.821" endtime="20210419 12:44:29.822"></status>
</kw>
<tags>
<tag>behavior-driven</tag>
</tags>
<status status="FAIL" starttime="20210419 12:44:29.816" endtime="20210419 12:44:29.823" critical="yes">No keyword with name 'FR1_Audits_LoggedNotification' found.</status>
</test>
<kw name="Close Browser" library="SeleniumLibrary" type="teardown">
<doc>Closes the current browser.</doc>
<status status="PASS" starttime="20210419 12:44:29.825" endtime="20210419 12:44:29.826"></status>
</kw>
<doc>FR1_01-01_CaptureAudit_AlternateScenario1</doc>
<status status="FAIL" starttime="20210419 12:44:29.314" endtime="20210419 12:44:29.826"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="2">Critical Tests</stat>
<stat pass="0" fail="2">All Tests</stat>
</total>
<tag>
<stat pass="0" fail="2">behavior-driven</stat>
</tag>
<suite>
<stat pass="0" fail="2" id="s1" name="FR1 01-01 CaptureAudit AS1">FR1 01-01 CaptureAudit AS1</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20210419 12:44:29.797" level="ERROR">Error in file 'C:\Users\Delwin.Horsthemke\PycharmProjects\Robot_ST_Templates\TestCases\Audits\FR1_01-01_CaptureAudit_AS1.robot' on line 4: Resource file '..\Resources\Audits\keywords.robot' does not exist.</msg>
<msg timestamp="20210419 12:44:29.801" level="ERROR">Error in file 'C:\Users\Delwin.Horsthemke\PycharmProjects\Robot_ST_Templates\TestCases\Audits\FR1_01-01_CaptureAudit_AS1.robot' on line 5: Resource file '..\Repository\Audits\FR1_Capture Audit_Keywords.robot' does not exist.</msg>
<msg timestamp="20210419 12:44:29.804" level="ERROR">Error in file 'C:\Users\Delwin.Horsthemke\PycharmProjects\Robot_ST_Templates\TestCases\Audits\FR1_01-01_CaptureAudit_AS1.robot' on line 6: Resource file '..\Resources\Audits\vars.robot' does not exist.</msg>
</errors>
</robot>
